pkg/config/runtime_setting.go

```diff
@@ -0,0 +1,68 @@
+/// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package config
+
+import (
+	"errors"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// RuntimeSettings registers all runtime editable config
+var RuntimeSettings = make(map[string](RuntimeSetting))
+
+// RuntimeSetting represents a setting that can be changed and read at runtime.
+type RuntimeSetting interface {
+	Get() (interface{}, error)
+	Set(v interface{}) error
+	Name() string
+	Description() string
+}
+
+func initRuntimeSettings() {
+	// Runtime-editable settings must be registered here to dynamically populate command-line information
+	registerRuntimeSetting(RuntimeSetting(ll))
+}
+
+// LogLevel wraps operations to change log level at runtime
```

Comment:
Could we move this to it's own file `runtime_setting_log_level.go` or something ?

Reply:
+1
We should have more runtime settings in the future and the `runtime_setting` part in `runtime_setting_log_level.go` definitely looks like a namespace to me, that's why I would even consider the runtime setting part having its proper sub-package in the `config` one. Feel free to ignore and only move it to a file like Maxime said if you prefer, we can move it later in another package when the needs will arise.

Reply:
I hesitated given the fact that log level implementation was rather small, but I agree it make sens to put it in a separate file.